src/quo2/components/avatars/group_avatar/view.cljs

```diff
@@ -7,24 +7,27 @@
 
 (def sizes
   {:icon      {:small  12
+               28      16
                :medium 16
                :large  20}
    :container {:small  20
+               28      28
                :medium 32
                :large  48}})
 
 ;; TODO: this implementation does not support group display picture (can only display default group
 ;; icon).
 (defn- view-internal
   [_]
-  (fn [{:keys [color size theme]}]
+  (fn [{:keys [color size theme icon-name]
```

Comment:
can we change `color` to `customization-color`? or is that a follow up?

Reply:
This PR already has a lot of changes related to the context-tag and its uses, adding even more to refactor surrounding components is too mucho IMO. I just touched the other components to add the required features, a separate issue for each one is required. I will create the issues

Reply:
Yep I agree, thanks!